packages/recommendations-vdom/src/types/Renderer.ts

```diff
@@ -0,0 +1,36 @@
+export type Pragma = (
+  type: any,
+  props: Record<string, any> | null,
+  ...children: ComponentChildren[]
+) => JSX.Element;
+export type PragmaFrag = any;
+
+type ComponentChild =
+  | VNode<any>
```

Comment:
and this?

Reply:
https://github.com/algolia/ui-components/blob/a2ac8d5bbb68c044d931e29b53b1ca832d174d83/packages/recommendations-vdom/src/types/Renderer.ts#L18-L21

Reply:
`VNode` is defined below.